src/modules/src/health.c

```diff
@@ -44,19 +44,20 @@
 #include "motors.h"
 #include "sensors.h"
 #include "pm.h"
+#include "autoconf.h"
 
 #include "static_mem.h"
 
-#ifndef DEFAULT_PROP_TEST_PWM_RATIO
-#  define DEFAULT_PROP_TEST_PWM_RATIO 0
+#ifndef CONFIG_MOTORS_DEFAULT_PROP_TEST_PWM_RATIO
```

Comment:
The `#ifndef` to set default value should not be needed, kbuild will define it with a default value already